scripts/install-emscripten.sh

```diff
@@ -33,13 +32,13 @@ if [ ! -d emsdk ]; then
 fi
 
 cd emsdk
-echo "Updating SDK base to ${EMSDK_VERSION} ..."
+echo "Updating SDK base to ${VERSION} ..."
```

Comment:
I have removed the `EMSDK_VERSION` and use `VERSION` now, the version that gets installed is always latest.